objectstorage/upload_part_request_response.go

```diff
@@ -95,6 +96,23 @@ type UploadPartResponse struct {
 	ETag *string `presentIn:"header" name:"etag"`
 }
 
+//
+// MultiPartUploadPart holds the details of Part that is uploaded
+//
+type MultiPartUploadPart struct {
```

Comment:
any modification here will be overwritten by the generated code, please move it under transfer folder.

Reply:
sure, done